cli/internal/destinations/postgresql/README.md

```diff
@@ -0,0 +1,20 @@
+# CloudQuery PostgreSQL Destination Plugin
+
+This destination plugin let's you sync data from any CloudQuery source to any PostgreSQL compatible database.
+
+Known supported databases:
+
+- PostgreSQL > v10
```

Comment:
```suggestion
- PostgreSQL > v10
```

Do we mean `>= 11`? Or `v10.0.1` is supported to?

Reply:
I think v10.0.1 should be supported as well. at least per my quick check locally. I didn't use any new shiny primitives.

Reply:
I think the only reason we supported 10 is because of Aurora which now supports 11 cc @bernays